README.md

```diff
@@ -11,11 +11,18 @@ built in Standalone ID Provider you can refer to the [Getting Started](https://d
 guide for details on how to obtain these. If you are using an OIDC provider you will need to obtain
 and ID Token in the usual way.
 
-When using the Standalone ID Provider the easiest way to use the D1 Storage client is through the
-`NewStorageClientWR` constructor. The resulting client will automatically refresh the user's access
-token when it expires. For specific code examples, see the [godocs](https://pkg.go.dev/github.com/cybercryptio/d1-client-go).
+The client can be configured with an option that provides an implementation of `credentials.PerRPCCredentials` to easily attach a token to every request. For convenience, we provide an implementation called `PerRPCToken` which can be any function that fetches a token, for example, after a call to the OIDC provider. There is also an implementation that can be used with the Standalone ID Provider:
 
-When using an OIDC provider you must use the `NewStorageClient` constructor and provide the ID Token as [gRPC metadata](https://pkg.go.dev/google.golang.org/grpc/metadata) via the context:
+```go
+client, _ := client.NewStorageClient(endpoint,
+		gclient.WithTransportCredentials(creds),
```

Comment:
Nothing is described anywhere about `gclient.WithTransportCredentials(creds)`, which is a bit confusing. I had to look at the code to understand why is was there. I think it would be a good idea to describe it bit, or omit it if not required.
The same goes for the `gclient.NewStandalonePerRPCToken` call

Reply:
Yea I removed `WithTransportCredentials` since it's not really relevant to show in the main README, and I added a clarification about `NewStandalonePerRPCToken`.